### DISCUSSION TITLE
Event videos are not streamed

### DISCUSSION THREAD

**Discussion by lpaolini** [blakeblackshear/frigate#14843] [Ask A Question]
I've noticed event videos are fully transferred to the browser using regular GET requests before they start playing, while live video is streamed over WebSocket and it starts playing immediately.

Could it be a configuration issue? Is it possible to stream event videos too?

Event video request:
http://x.x.x.x:5000/vod/cam1/start/1730966400/end/1730969254/seg-1-v1.m4s

Live video request:
ws://x.x.x.x:5000/live/mse/api/ws?src=cam1

**Comment by NickM-27** [comment]
Event videos are streamed using HLS

**Comment by lpaolini** [comment]
I see, but this is the behaviour I get while reviewing events, even on latest Frigate 0.15.0 beta 2.
[video](https://www.youtube.com/shorts/cwpqmp06r58)
